OxfordBodleian/Ullendorff/BDLeu7.xml

```diff
@@ -34,59 +37,212 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                        </msItem>
+                        <msItem xml:id="ms_i6">
+                            <locus from="1r124" to="1r157"/>
+                            <title type="complete" ref="LIT5696MagicPrayer"/>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez">
+                                <hi rend="rubric">መስቀል፡ ሐ<gap reason="lost" unit="chars" quantity="1"/>ር፡ ወጥቅም፡ መስቀል፡ ሐፁር፡ ለክርስቲያን፡ መስቀል፡ ሐፁር፡ ውስተ፡
+                                    <gap reason="lost" unit="chars" quantity="1"/>ዳ፡ ዕልዋን፡ መስቀል፡ ሐፁር፡</hi> በቤተ፡ ክርስቲያን፡ መስቀል፡ ሐፁር፡ ለትሑታን፡ መስቀለ፡ ሐፁር፡
+                            </incipit>
+                            <note>The explicit is poorly readable due to the condition of the parchment.</note>
+                        </msItem>
                     </msContents>
                     
                     <physDesc>
-                        <objectDesc form="Codex">
+                        <objectDesc form="Scroll">
                             <supportDesc>
                                 <support>
                                     <material key="parchment"/>
                                 </support>
                                 <extent>
-                                    <measure unit="strip">3</measure>
+                                    <note>Consists of <measure unit="strips">3</measure> strips sewn together.</note>
                                     <dimensions type="outer" unit="mm">
                                         <height>1245</height>
                                         <width>100-112</width>
                                     </dimensions>
+                                    <dimensions type="outer" unit="mm" xml:id="strip1">
+                                        <height>545</height>
+                                        <width>112</width>
+                                    </dimensions>
+                                    <dimensions type="outer" unit="mm" xml:id="strip2">
+                                        <height>498</height>
+                                        <width>110</width>
+                                    </dimensions>
+                                    <dimensions type="outer" unit="mm" xml:id="strip3">
+                                        <height>236</height>
+                                        <width>100-104</width>
+                                    </dimensions>
                                     <measure type="weight" unit="g">39</measure>
-                                </extent> 
+                                </extent>
+                              <condition key="good">The manuscript is in a mediocre state of preservation. 
+                              The parchment is crumpled due to rolling and is darkened toward the lower border; subsequently, the text is barely legible.
+                              Some letters are illegible due to material damage.
```

Comment:
I would add 'Small areas of loss and tears are present, especially along the upper and lower margins'. Or something like that. :)